### PR TITLE
#31 user story welcoming block

### LINKED ISSUE #31: Guest landing. Welcoming block
**As a Guest, I want to see the welcoming block (screen) of the home page, so that I can find out the general information about Space2Study platform and register an account**

_Acceptance criteria:_
1.	When a Guest enters the home page, then he/she sees the home page welcoming block.
The Mockups:
•	for Desktop/Tablet
Image

2.	The home page welcoming block contains:
–	the CRUD welcoming text from above. The welcoming text consists of a title (“The secret of getting ahead is getting started.”) and a description (“An educational platform that offers professional courses, empowering students with essential skills for career success.”);
–	the “Get started for free” button below the CRUD welcoming text;
–	the static world map image below the “Get started for free” button;
–	the list with four phrases from the right of the static world map.
3.	The “Get started for free” button is clickable. When a Guest clicks on the button, then the system scrolls down the home page to the “What can you do” block.
4.	When a Guest hovers over the “Get started for free” button, then the color changes.
5.	The list with four phrases from the right of the world map is active and the phrases in the list have the order as shown on the mockup.
-	each phrase (item) of the list is clickable;
-	a Guest can open a description to any of four items in the list;
-	the first upper item description is opened by default;
-	when a Guest clicks on any other item of the list, then the description to the clicked item under the phrase appears and the description to the previous opened item closes;
-	the description and the clicked phrase change their text and background color;
-	when the description of one of the items opened, then the items which are below move down.
6.	Each phrase has the description:
-	The “Flexible Location” phrase has the “Space2Study platform has no boundaries and can be used anywhere in the world, all you need is aspiration.” text above;
-	The “Individual Time” phrase has the “Space2Study platform provides the opportunity to study lessons at any time convenient for you, pause and continue again.” text above;
-	The “Free Choice of Tutors” phrase has the “Space2Study platform has a wide range of criteria and will help you find the best tutor for education.” text above;
-	The “Digital Communication” phrase has the “Space2Study platform offers extensive opportunities for meaningful communication not only with tutors but also with students.” text above.
7.	When a Guest hovers over a phrase, then the background area appears.
•	for Mobile
Image

8.	The carousel with four items with description below the world map is active and the items in the carousel have the order as shown on the mockup.
-	a Guest can swipe each phrase (item) of the carousel.

### PR DESCRIPTION
**Pull request by mistercapxx**
Screenshot 2024-07-24 at 13 18 51
Screenshot 2024-07-24 at 13 18 33
Worked on button animation, on reaction if we click on it. Worked with mobile adaptive part. Made description block move while world map remains on same position.

### CODE REVIEW

**Review by CallMeMaverick — COMMENTED** [review]
image

Two issues in sonar, perhaps those should be addressed

**Review by markgol777 — CHANGES_REQUESTED** [review]
The unit testing for SearchInput was not part of the task

File: src/components/accordion/Accordion.styles.ts
Comment (miller-113) [inline src/components/accordion/Accordion.styles.ts:1]: Need to correct formatting
Screenshot 2024-07-26 at 15 41 34
 
Screenshot 2024-07-26 at 15 42 23

File: src/components/accordion/Accordions.tsx
Comment (miller-113) [inline src/components/accordion/Accordions.tsx:1]: Why in this case you using destructurization? 
Screenshot 2024-07-26 at 15 43 43

File: src/components/accordion-with-image/AccordionWithImage.styles.ts
Comment (miller-113) [inline src/components/accordion-with-image/AccordionWithImage.styles.ts:1]: formatting

File: src/components/accordion-with-image/AccordionWithImage.tsx
Comment (miller-113) [inline src/components/accordion-with-image/AccordionWithImage.tsx:1]: formatting. Too much space between return.

File: src/components/carousel-with-image/CarouselWithImage.styles.ts
Comment (miller-113) [inline src/components/carousel-with-image/CarouselWithImage.styles.ts:1]: formatting

File: src/components/carousel-with-image/CarouselWithImage.tsx
Comment (miller-113) [inline src/components/carousel-with-image/CarouselWithImage.tsx:1]: formatting

File: src/containers/guest-home-page/styles/Welcome.styles.ts
Comment (miller-113) [inline src/containers/guest-home-page/styles/Welcome.styles.ts:1]: formatting and u left commented line.

File: src/pages/guest-home-page/GuestHome.tsx
Comment (miller-113) [inline src/pages/guest-home-page/GuestHome.tsx:1]: Screenshot 2024-07-26 at 15 52 35

formatting, it's must look line in the screenshot

**Review by miller-113 — COMMENTED** [review]
Files need reformatting. The best way to achieve this using prettier

**Review by Yaroslav-Yakovlev — APPROVED** [review]

**Review by miller-113 — APPROVED** [review]
👍

**Review by CallMeMaverick — APPROVED** [review]

**Review by markgol777 — APPROVED** [review]

**Review by nrevutska — APPROVED** [review]

**Pull request merged**